### PR TITLE
NUTCH-2516 Hadoop imports use wildcards

### PR DESCRIPTION
**Pull request by lewismc**
Large but simple patch which addresses https://issues.apache.org/jira/projects/NUTCH/issues/NUTCH-2516

### CODE REVIEW

**Comment by lewismc** [comment]
I would like to commit by EoB today if possible. We can then shift priority to #299 I will that through with @kpm1985 

Any comments folks?

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,8 @@ build/
 runtime/
 logs/
 /bin/
+.classpath
+.naivebayes-model.crc
+.project
+ivy/ivy-2.3.0.jar
```
Comment (sebastian-nagel) [inline .gitignore]: What about ivy/ivy-2.4.0.jar which has been committed in f9981d4? It's supposed to be "installed" by `ant ivy-download`.

Comment (lewismc) [reply .gitignore]: Thanks @sebastian-nagel I've updated the PR to also prevent ```ivy-2.4.0.jar``` from being added to SCM.

Comment (sebastian-nagel) [reply .gitignore]: +1 Maybe also remove this file to avoid confusions?

Comment (sebastian-nagel) [reply .gitignore]: and please go forward and commit, thanks!

**Pull request merged**